src/main/java/pl/coderstrust/accounting/logic/pdf/PdfSet.java

```diff
@@ -0,0 +1,104 @@
+
+package pl.coderstrust.accounting.logic.pdf;
+
+import org.apache.pdfbox.pdmodel.common.PDRectangle;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import pl.coderstrust.accounting.model.Invoice;
+import pl.coderstrust.accounting.model.InvoiceEntry;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+public class PdfSet {
+
+  private static final PDRectangle PAGE_SIZE = new PDRectangle(841.8898F, 595.27563F);
+  private static final float MARGIN = 50;
+  private static final boolean IS_LANDSCAPE = true;
+
+  private static final PDFont TEXT_FONT = PDType1Font.HELVETICA;
+  private static final float FONT_SIZE = 10;
+
+  private static final float ROW_HEIGHT = 30;
+  private static final float CELL_MARGIN = 4;
+
+  public static Table createContent(Invoice invoice) throws IOException {
+
```

Comment:
This method is long and i think we can split this into some private methods

Reply:
done